Dockerfile.local

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.11-alpine
+RUN apk add --update --no-cache ca-certificates git
+RUN go get -d github.com/kubernetes-sigs/aws-iam-authenticator/cmd/aws-iam-authenticator
+RUN cd $GOPATH/src/github.com/kubernetes-sigs/aws-iam-authenticator && \
+    git checkout 981ecbe && \
+    go install ./cmd/aws-iam-authenticator
+
+FROM alpine:3.7
+RUN apk add --no-cache ca-certificates libc6-compat
```

Comment:
libc6-compat is not necessary here since we don't use delve.
Certificates can be copied from the previous layer

Reply:
But there is no delve anywhere around this docker image